lib/features/qr_scanner/cubit/camera_cubit.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:givt_app_kids/features/organisation/models/organisation.dart';
+part 'camera_state.dart';
+
+class CameraCubit extends Cubit<CameraState> {
```

Comment:
do we emit CameraLoading state somewhere?

Reply:
Nope, I used it, but with changes I forgot to delete it. Its gone now.